src/Dfc.CourseDirectory.WebV2/Behaviors/TempDataKeys.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Dfc.CourseDirectory.WebV2.Behaviors
```

Comment:
This namespace is more for MediatR cross-cutting stuff. Could go in root `WebV2` namespace, for me.